packages/server/src/sync/service.ts

```diff
@@ -33,7 +31,7 @@ export class SyncService extends Service {
   }
 
   async setup() {
-    let config = this.config.current.sync
+    let config = undefined
```

Comment:
Remove this line and put the for loop inside the try-catch?!

Reply:
Good point